src/HID-Settings.h

```diff
@@ -111,12 +111,17 @@ THE SOFTWARE.
 
 #define ATTRIBUTE_PACKED  __attribute__((packed, aligned(1)))
 
+#define USB_EP_SIZE                 EPX_SIZE
+#define EP_TYPE_INTERRUPT_IN        USB_ENDPOINT_TYPE_INTERRUPT | USB_ENDPOINT_IN(0);
+#define EP_TYPE_INTERRUPT_OUT       USB_ENDPOINT_TYPE_INTERRUPT | USB_ENDPOINT_OUT(0);
+
+#if defined(ARDUINO_API_VERSION)
+#include "api/PluggableUSB.h"
+#define EPTYPE_DESCRIPTOR_SIZE		unsigned int
```

Comment:
why is it unsigned int here? If so, please use uint16_t

Reply:
These changes are based on similar changes the Arduino folks made to their MIDIUSB library having to do with ARDUINO_API. I suspect the idea is to have the same USB API for AVR and SAMD. unsigned int will be 16 bits for AVR and 32 bits for SAMD. I have no direct information on the purpose for the ARDUINO_API changes.

https://github.com/arduino-libraries/MIDIUSB/commit/80b6fac501702a268d7f9ab9702333c5d059e059

Reply:
Alright, that makes sense. Was not aware that its 32 bit on SAMD